kubernetes/chart/zulip/.gitignore

```diff
@@ -0,0 +1,2 @@
+./charts
+Chart.lock
```

Comment:
any reason for this? I think it is expected to commit this. See https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack for reference

Reply:
No reason, I removed this from the gitignore and added Chart.lock